perun-base/src/main/resources/perun-roles.yml

```diff
@@ -7949,6 +7953,8 @@ perun_roles_management:
       - GROUPOBSERVER
     assignable_to_attributes: true
     display_name: "Group admin"
+    receive_notifications:
+      - GROUP
```

Comment:
Why not use 'Group' and 'Vo' to be consistent with `primary_object` etc..? Is there any specific reason that I'm missing for it to be all caps?

Reply:
no specific reason, just personal preference for upper case list elements. changed here and in ansible template docs.